More2Life/FeedViewController.swift

```diff
@@ -37,7 +37,7 @@ class FeedViewController: UIViewController {
             
             let fetchRequest: NSFetchRequest<FeedItem> = FeedItem.fetchRequest()
             fetchRequest.predicate = NSPredicate(format: "%K == true", #keyPath(FeedItem.isActive))
-            fetchRequest.sortDescriptors = [NSSortDescriptor(key: #keyPath(FeedItem.index), ascending: false)]
+            fetchRequest.sortDescriptors = [NSSortDescriptor(key: #keyPath(FeedItem.identifier), ascending: false)]
```

Comment:
This is weird. We won't be able to change to order of things.

Reply:
Derp. I was thinking the identifier is a number but it is not - it's a string. This won't work here. 

Does a date work as a sort descriptor? If we added a createdAt date to all the feed items could we use that here?

Reply:
Yes. That would work great.